addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -31,6 +31,11 @@
 
 .o_ChatterTopbar {
 
+    /*
+    Allow topbar to be used with personalized background.
+    E.g. in documents, the topbar should have a black border top given by its container.
+     */
+    background: none;
```

Comment:
removing this style rule doesn't change anything, because it looks like the inset black color is above of the chatter topbar on Document